source/frontend/src/components/common/Section/SectionField.tsx

```diff
@@ -66,4 +66,7 @@ export const StyledCol = styled(Col)`
 
 export const StyledFieldLabel = styled.label`
   font-weight: bold;
+  span {
+    position: absolute;
+  }
```

Comment:
again - this seems a broad change (like there are 89 changed snapshots)
Similarly to the other PR perhaps we should discuss as a team

Reply:
sure, the impact it's barely noticeable but I am ok with a quick call.

Reply:
aligned all the columns a little more to the right, since it's better than the alternatives.